### PR TITLE
- Refactor: added names for powerups and updated some old.

### PR DESCRIPTION
**Pull request by NoOneBlood**
- AI: there is printing error message in conosole added (instead of dassert)
- Updates for Player Control modern type

### CODE REVIEW

File: source/blood/src/player.cpp [hunk 1]
```diff
@@ -625,46 +660,44 @@ void powerupInit(void)
...
 int powerupToPackItem(int nPowerUp)
 {
-    const int jumpBoots = 15;
-    const int divingSuit = 18;
-    const int crystalBall = 21;
-    const int beastVision = 25;
+    //const int jumpBoots = kPwUpJumpBoots;
+    //const int divingSuit = kPwUpDivingSuit;
+    //const int crystalBall = kPwUpCrystalBall;
+    //const int beastVision = 25;
```
Comment (CommonLoon102) [inline source/blood/src/player.cpp]: I think these can be deleted instead of commented out.

File: source/blood/src/player.cpp [hunk 2]
```diff
@@ -831,20 +864,14 @@ void playerCorrectInertia(PLAYER* pPlayer, vec3_t const *oldpos)
...
 void playerResetPowerUps(PLAYER* pPlayer)
 {
-    const int jumpBoots = 15;
-    const int divingSuit = 18;
-    const int crystalBall = 21;
-    const int beastVision = 25;
+    //const int jumpBoots = kPwUpJumpBoots;
+    //const int divingSuit = kPwUpDivingSuit;
+    //const int crystalBall = kPwUpCrystalBall;
+    //const int beastVision = 25;
```
Comment (CommonLoon102) [inline source/blood/src/player.cpp]: I think these can be deleted instead of commented out.

**Comment by NoOneBlood** [comment]
Yeah, i maybe forget it. Will do next time

File: source/blood/src/view.cpp [hunk 3]
```diff
@@ -2476,6 +2476,8 @@ void viewProcessSprites(int32_t cX, int32_t cY, int32_t cZ, int32_t cA, int32_t
             if (powerupCheck(gView, kPwUpBeastVision) > 0) pTSprite->shade = -128;
 
             if (IsPlayerSprite((spritetype *)pTSprite)) {
+              viewApplyDefaultPal(pTSprite, pSector);
```
Comment (CommonLoon102) [inline source/blood/src/view.cpp]: Need two more spaces for indent :)

**Pull request merged**